lcm/service/lcm/container_helper_extensions.go

```diff
@@ -35,7 +35,7 @@ func extendLearnerContainer(learner *v1core.Container, req *service.JobDeploymen
 		// TODO!
 	}
 
-	extCmd := "export PATH=" + learnerEntrypointFilesPath + ":$PATH; chmod +x " + learnerEntrypointFilesPath + "/*.sh; "
+	extCmd := "export PATH=/:$PATH; cp " + learnerEntrypointFilesPath + "/*.sh /; chmod +x /*.sh; "
```

Comment:
is there any better candidates besides the root to copy that file(s) to?  In DLaaS we copy to /usr/local/bin/train.sh, so could you copy it(them) there?

Reply:
Sure

Reply:
@sboagibm Changed the train.sh path to `/usr/local/bin/train.sh` and the travis build is passed.